### PR TITLE
Increased Inclusiveness of the CoC

### PR DESCRIPTION
**Pull request by Stratege**
- Removed implicit endorsements of harassment.
- Added emphasis on being nice to each other.
- Removed overly specific language, which only serves to single out groups.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -33,22 +34,20 @@ Harassment includes, but is not limited to:
...
+- Reasonable critic of negative behavior.
 
-### Diversity Statement
+### Statement of Welcomeness
 
 We encourage everyone to participate and are committed to building a community for all. Although we will fail at times, we seek to treat everyone both as fairly and equally as possible. Whenever a participant has made a mistake, we expect them to take responsibility for it. If someone has been harmed or offended, it is our responsibility to listen carefully and respectfully, and do our best to right the wrong.
 
-Although this list cannot be exhaustive, we explicitly honor diversity in age, gender, gender identity or expression, culture, ethnicity, language, national origin, political beliefs, profession, race, religion, sexual orientation, socioeconomic status, and technical ability. We will not tolerate discrimination based on any of the protected
-characteristics above, including participants with disabilities.
+We explicitly honor everyone. We will not tolerate discrimination based on anything.
```
Comment (asiekierka) [inline index.md]: This is very vague. Does "everyone" and "anything" include things such as murder?

File: index.md [hunk 2]
```diff
@@ -33,22 +34,20 @@ Harassment includes, but is not limited to:
...
-
-Our open source community prioritizes marginalized people’s safety over privileged people’s comfort. We will not act on complaints regarding:
-
 - ‘Reverse’ -isms, including ‘reverse racism,’ ‘reverse sexism,’ and ‘cisphobia’
-- Reasonable communication of boundaries, such as “leave me alone,” “go away,” or “I’m not discussing this with you”
-- Refusal to explain or debate social justice concepts
-- Communicating in a ‘tone’ you don’t find congenial
-- Criticizing racist, sexist, cissexist, or otherwise oppressive behavior or assumptions
 
+Our open source community wants everyone to feel welcome, therefor we will not act on complaints regarding:
```
Comment (asiekierka) [inline index.md]: Therefore

File: index.md [hunk 3]
```diff
@@ -13,14 +13,15 @@ Our open source community strives to:
...
 
 ## Definitions
 
 Harassment includes, but is not limited to:
 
-- Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, race, age, regional discrimination, political or religious affiliation
-- Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment
-- Deliberate misgendering. This includes deadnaming or persistently using a pronoun that does not correctly reflect a person's gender identity. You must address people by the name they give you when not addressing them by their username or handle
+- Offensive or Unwelcome comments in general, when asked to stop please do so.
```
Comment (asiekierka) [inline index.md]: "You're that troll from /baph/!"
"WHAT AN UNWELCOME COMMENT PLEASE STOP"

**Comment by asiekierka** [comment]
The "Be Forgiving/Thoughtful" are good additions. The rest has gaping holes waiting for abuse.

**Comment by bkeepers** [comment]
Thanks for your feedback. See https://github.com/todogroup/opencodeofconduct/issues/84.

**Pull request closed without merging**